pkg/apis/migration/v1alpha1/resource.go

```diff
@@ -8,7 +8,7 @@ const (
 	TouchAnnotation  = "touch"
 	FinalizerEnabled = false
```

Comment:
Re-enabling finalizers caused the same issues at this point, not allowing to remove the migPlan.  Will leave it as is for now.